### PR TITLE
Fix DOS 5 content issues discovered during testing

### PR DESCRIPTION
**Pull request by bjgill**
Trello: https://trello.com/c/vkiTHroy/417-3-test-supplier-signup-journey-on-preview-whole-team

This fixes most of the issues identified to date in [the spreadsheet of errors](https://docs.google.com/spreadsheets/d/1JCnzLu-XpoqRYIrqA12hlZO4IdkvllI_kw-tqLCuTzk)

### CODE REVIEW

**Review by phantas — COMMENTED** [review]
Probably one for @NoraGDS , but under Briefs for DOS5, specifically `questions/briefs/researchAddress.yml`, there is this line:

> - a description of the research environment, such as whether the research will happen in a lab, at a participant’s home,

Do we want to replace lab with research studio here too?

File: frameworks/digital-outcomes-and-specialists-5/questions/declaration/subcontracting.yml [hunk 1]
```diff
@@ -1,5 +1,5 @@
 name: Subcontractors or consortia
-question: "Please confirm whether you will provide Digital Outcomes and Specialists&nbsp;5 services:"
+question: "Can you confirm whether you will provide Digital Outcomes and Specialists&nbsp;5 services:"
```
Comment (lfdebrux) [inline frameworks/digital-outcomes-and-specialists-5/questions/declaration/subcontracting.yml:2]: There are lots of uses of 'please' in the declaration questions, should we be changing those as well?

**Review by phantas — COMMENTED** [review]
On `questions/services/labAccessibility.yml`:

> Describe the ways in which your lab is accessible or inaccessible, such as wheelchair access or hearing loops provided.

Lab should probably be replaced by studio

**Review by phantas — COMMENTED** [review]
On `questions/services/labPrice.yml`:

> question: What is the minimum amount of time your lab can be booked for and how much does it cost (excluding VAT)?

and

> message: 'Enter the cost of using your lab.'

lab should probably be replaced by studio

File: frameworks/digital-outcomes-and-specialists-5/manifests/declaration.yml [hunk 2]
```diff
@@ -121,7 +121,7 @@
   description: |
     You must be able to truthfully answer ‘yes’ to every question on this page for your application to be considered eligible.
 
-    If you can’t answer ‘yes’ to every question in this section, it is unlikely your application will be accepted.
+    If you cannot answer ‘yes’ to every question in this section, it is unlikely your application will be accepted.
```
Comment (lfdebrux) [inline frameworks/digital-outcomes-and-specialists-5/manifests/declaration.yml:124]: There are quite a few other uses of 'can't' in the DOS5 content:

```
gds5506:digital-outcomes-and-specialists-5 laurencedebruxelles$ rg -i "can['‘’]t" -o
manifests/declaration.yml
22:can’t
124:can’t

manifests/edit_brief.yml
74:can’t
111:can’t
144:can’t

manifests/edit_submission.yml
11:can’t
24:can’t

questions/brief-responses/dayRate.yml
20:can't

questions/briefs/outcome.yml
4:can’t

questions/briefs/contractLength.yml
3:can’t

questions/briefs/startDate.yml
4:can’t
```
```

File: frameworks/digital-outcomes-and-specialists-5/manifests/declaration.yml [hunk 3]
```diff
@@ -162,7 +162,6 @@
   questions:
     - multiqSubcontracting
     - employmentStatus
-    - outsideIR35
```
Comment (lfdebrux) [inline frameworks/digital-outcomes-and-specialists-5/manifests/declaration.yml:165]: 🙌

**Comment by phantas** [comment]
On `questions/services/labSize.yml:`

>name: How many people can the lab accommodate?
>questions/services/labSize.yml:  How many people can the lab accommodate?

lab should probably be replaced by studio

**Comment by phantas** [comment]
On `questions/services/lot.yml`:

> User research studios sometimes have multiple labs, or rooms. Labs usually include a device to test a service
> and a way to observe the session. Provide information on each lab you want buyers to use.

lab should probably be replaced by studio or research studio

**Comment by NoraGDS** [comment]
> Probably one for @NoraGDS , but under Briefs for DOS5, specifically `questions/briefs/researchAddress.yml`, there is this line:
> 
> > * a description of the research environment, such as whether the research will happen in a lab, at a participant’s home,
> 
> Do we want to replace lab with research studio here too?

yes, please change to 'a research studio'.

**Comment by NoraGDS** [comment]
> On `questions/services/lot.yml`:
> 
> > User research studios sometimes have multiple labs, or rooms. Labs usually include a device to test a service
> > and a way to observe the session. Provide information on each lab you want buyers to use.
> 
> lab should probably be replaced by studio or research studio

Second mention of labs should be replaced with: Research studios
Third mention of lab should be replaced with: research studio

**Review by phantas — CHANGES_REQUESTED** [review]
Just a few sprinkled changes of "lab" to "studio" or "research studio" (see previous comments) and a clarification on the spreadsheet regarding row 22 and row 27 (about question 10, I believe)

**Comment by NoraGDS** [comment]
> On `questions/services/labSize.yml:`
> 
> > name: How many people can the lab accommodate?
> > How many people can the lab accommodate?
> 
> lab should probably be replaced by studio

Please replace both with: research studio

File: frameworks/digital-outcomes-and-specialists-5/questions/declaration/subcontracting.yml (hunk 1 above)
Comment (NoraGDS) [reply frameworks/digital-outcomes-and-specialists-5/questions/declaration/subcontracting.yml:2]: Yes, all they should all start with 'Can you' - if it's doable please.

**Comment by NoraGDS** [comment]
> On `questions/services/labPrice.yml`:
> 
> > question: What is the minimum amount of time your lab can be booked for and how much does it cost (excluding VAT)?
> 
> and
> 
> > message: 'Enter the cost of using your lab.'
> 
> lab should probably be replaced by studio

Please replace with: research studio

**Comment by NoraGDS** [comment]
> On `questions/services/labAccessibility.yml`:
> 
> > Describe the ways in which your lab is accessible or inaccessible, such as wheelchair access or hearing loops provided.
> 
> Lab should probably be replaced by studio

Please replace with: research studio

File: frameworks/digital-outcomes-and-specialists-5/manifests/declaration.yml (hunk 2 above)
Comment (lfdebrux) [reply frameworks/digital-outcomes-and-specialists-5/manifests/declaration.yml:124]: Fixed in #642

File: frameworks/digital-outcomes-and-specialists-5/questions/declaration/subcontracting.yml (hunk 1 above)
Comment (bjgill) [reply frameworks/digital-outcomes-and-specialists-5/questions/declaration/subcontracting.yml:2]: Handled in https://github.com/alphagov/digitalmarketplace-frameworks/pull/644

**Review by phantas — APPROVED** [review]
🚀

**Pull request merged**